### PR TITLE
LPD-26035 Horizontal Scroll Not Working Properly After Screen Resize

### PR DESCRIPTION
**Pull request by veroglez**
This PR does not have tests because the changes are only CSS adjustments

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss [hunk 1]
```diff
@@ -112,6 +111,7 @@ html#{$cadmin-selector} .cadmin .page-editor__sidebar {
 		display: flex;
 		flex-direction: column;
 		font-size: 0.875rem;
+		height: calc(100vh - --control-menu-container-height);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss:114]: Does this works? should it be like `var(--control-menu-container-height)`?

**Comment by victorg1991** [comment]
Hey @veroglez  this breaks the zoomed version of the page editor :'(

https://github.com/liferay-echo/liferay-portal/assets/11654230/9418f35d-c180-4f33-8cf4-d758b73602df

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss (hunk 1 above)
Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss:114]: I think I accidentally deleted something when editing the commit. Sorry :sweat_smile:

**Comment by victorg1991** [comment]
Nicceeee :clap:

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
@veroglez you have to add this commit: https://github.com/liferay-echo/liferay-portal/pull/15813/commits/21d33fb02abbe30882f51b522c47e0c20338e4b3 for the sf to pass

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**